### PR TITLE
Sebastians GUI

### PR DESCRIPTION
**Pull request by Sebastianchr22**

### CODE REVIEW

File: SensumUdred_UI/src/data/UICitizen.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+package data;
+
+import common.Gender;
+import common.ICase;
+import common.ICitizen;
+import common.RelationshipStatus;
+
+/**
+ *
+ * @author Sebas
```
Comment (larsbjn) [inline SensumUdred_UI/src/data/UICitizen.java]: Authors

File: SensumUdred_UI/src/data/UICitizen.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
```
Comment (larsbjn) [inline SensumUdred_UI/src/data/UICitizen.java]: License header need remove

File: SensumUdred_Persistence/src/persistence/DataCitizen.java [hunk 3]
```diff
@@ -32,11 +32,6 @@
      */
     private Gender gender;
     
-    /**
-     * The active for the citizen, can be null
-     */
-    private DataCase activeCase;
-
```
Comment (krongrah) [inline SensumUdred_Persistence/src/persistence/DataCitizen.java]: Why is this being deleted

File: SensumUdred_UI/src/data/UICitizenData.java [hunk 4]
```diff
@@ -0,0 +1,89 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
```
Comment (larsbjn) [inline SensumUdred_UI/src/data/UICitizenData.java]: License header

File: SensumUdred_UI/src/data/UICitizenData.java [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+ */
+package data;
+
+import common.ICaseWorker;
+import common.ICitizen;
+import common.ICitizenData;
+import java.util.Date;
+
+/**
+ *
```
Comment (larsbjn) [inline SensumUdred_UI/src/data/UICitizenData.java]: Author

File: SensumUdred_UI/src/ui/GUI.java [hunk 6]
```diff
@@ -5,32 +5,44 @@
...
 import javafx.fxml.FXMLLoader;
 import javafx.scene.Parent;
 import javafx.scene.Scene;
+import javafx.scene.image.Image;
 import javafx.stage.Stage;
 
 /**
  *
- * @author fsr19
+ * @author Sebas
```
Comment (larsbjn) [inline SensumUdred_UI/src/ui/GUI.java]: Author

File: SensumUdred_UI/src/data/UICitizenData.java [hunk 7]
```diff
@@ -0,0 +1,89 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
```
Comment (krongrah) [inline SensumUdred_UI/src/data/UICitizenData.java]: remove header

File: SensumUdred_Persistence/src/persistence/DataCitizen.java [hunk 8]
```diff
@@ -52,15 +47,7 @@ public DataCitizen(ICitizen citizen) {
...
         relationshipStatus = citizen.getRelationshipStatus();
-        activeCase = null;
-    }
-    
-    /**
-     * Sets the active case for the citizen
-     * @param _case the given case
-     */
-    public void setActiveCase(DataCase _case) {
-        activeCase = _case;
```
Comment (krongrah) [inline SensumUdred_Persistence/src/persistence/DataCitizen.java]: why deletion?

File: SensumUdred_UI/src/data/UICitizen.java [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+ * and open the template in the editor.
+ */
```
Comment (krongrah) [inline SensumUdred_UI/src/data/UICitizen.java]: remove header

File: SensumUdred_UI/src/data/UICitizen.java (hunk 1 above)
Comment (krongrah) [inline SensumUdred_UI/src/data/UICitizen.java]: authors

File: SensumUdred_UI/src/data/UICitizen.java [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+import common.Gender;
+import common.ICase;
+import common.ICitizen;
+import common.RelationshipStatus;
+
+/**
+ *
+ * @author Sebas
+ */
+public class UICitizen implements ICitizen {
```
Comment (AndreasBoegh) [inline SensumUdred_UI/src/data/UICitizen.java]: Javadoc

File: SensumUdred_UI/src/data/UICitizenData.java [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+import common.ICaseWorker;
+import common.ICitizen;
+import common.ICitizenData;
+import java.util.Date;
+
+/**
+ *
+ * @author Sebas
+ */
+public class UICitizenData implements ICitizenData {
```
Comment (AndreasBoegh) [inline SensumUdred_UI/src/data/UICitizenData.java]: Javadoc

File: SensumUdred_UI/src/ui/GUI.java [hunk 12]
```diff
@@ -5,32 +5,44 @@
...
  */
-public class SensumUdred_UI extends Application {
+public class GUI extends Application implements IUI {
```
Comment (AndreasBoegh) [inline SensumUdred_UI/src/ui/GUI.java]: Javadoc

**Pull request merged**